### PR TITLE
Feature/3746 global view protocol gas data

### PR DESCRIPTION
**Pull request by cw-cvp**

### CODE REVIEW

File: src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/ProtocolGasDataRows.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+          />
+        </div>
+      }
+    </>
+  )
+}
+
+export default ProtocolGasDataRows
+
+const mapProtocolGasDataToRows = (data = []) => {
```
Comment (Qb10) [inline src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/ProtocolGasDataRows.js]: i would move this to its own file in utils/selectors to keep align with the rest of the codebase

**Review by Qb10 — CHANGES_REQUESTED** [review]

File: src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/ProtocolGasDataRows.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    'Gas Type Code',
+    'Cylinder ID',
+    'Vendor ID',
+    'Expiration Date',
+  ];
+
+  useEffect(() => {
+    const fetchProtocolGas = async () => {
+      setLoading(true)
+      const resp = await getQAProtocolGas(locationId, id)
```
Comment (addisgmichael) [inline src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/ProtocolGasDataRows.js:30]: can you add a validation to the API response to check if the response is a success, and add a handler if not.  Also this way when you run the associated unit test, it won't complain.

File: src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/ProtocolGasDataRows.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+import React, { useState, useMemo, useEffect } from "react";
+import { Button } from "@trussworks/react-uswds";
+
+import { getQAProtocolGas } from "../../../utils/api/qaCertificationsAPI";
+import QADataTableRender from "../../QADataTableRender/QADataTableRender";
+import { Preloader } from "@us-epa-camd/easey-design-system";
+
+const tableLabel = 'Protocol Gas'
+
+const ProtocolGasDataRows = ({
```
Comment (addisgmichael) [inline src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/ProtocolGasDataRows.js:12]: also this component is supposed to be reusable for other test type groups being selected so better to move it under the folder qaDatatablesContainer

File: src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/QALinearitySummaryExpandableRows.js [hunk 4]
```diff
@@ -1,405 +1,32 @@
...
-export { mapStateToProps };
+    <>
+      <LinearitySummaryDataRows
+        user={user}
+        nonEditable={nonEditable}
+        locationSelectValue={locationSelectValue}
+        data={data}
+      />
+      <ProtocolGasDataRows
+        user={user}
```
Comment (addisgmichael) [inline src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/QALinearitySummaryExpandableRows.js:23]: i don't think we need a separate component to render both LinearitySummaryDataRows and ProtocolGasDataRows as you can just import and throw it in QALinearitySummaryExpandableRows

**Pull request closed without merging**